modules/gitops-release-notes-1-16-0.adoc

```diff
@@ -0,0 +1,159 @@
+
+* Before this update, during an upgrade, the `argocd-redis-ha-configmap`, `argocd-redis-ha-health-configmap`, and the Redis HA `StatefulSet` were not correctly updated. This led to Redis HA pods encountering an `AUTH` error.
+With this update, the {gitops-shortname} Operator correctly updates the Redis HA configmaps and `StatefulSet` during an upgrade process. As a result, Redis HA pods are now prevented from entering an `AUTH` error state post-upgrade. link:https://issues.redhat.com/browse/GITOPS-5975[GITOPS-5975]
+
+* Before this update, any changes to the `serviceAccountName` and `serviceAccount` fields in the Redis deployment were not reconciled by the {gitops-title} Operator. With this update, this issue is fixed by ensuring that any unintended changes to these fields are reset to their expected value, `<argocd-instance-name>-argocd-redis`. link:https://issues.redhat.com/browse/GITOPS-6032[GITOPS-6032]
+
+* Before this update, Argo CD relied solely on the `sub claim` for user identification, which could be non-deterministic with Dex and cause unexpected RBAC policy failures.With this update, Argo CD identifies users in the following order:
++
+--
+** First, it checks the `federated_claims.user_id` when Dex is the identity provider.
+** Second, if federated claims are unavailable or empty, it falls back to the sub claim.
+--
++
+--
+With this update, this issue is fixed. This change ensures RBAC policies are based on actual user identifiers (such as email addresses) rather than encoded values, improving clarity and consistency.
+--
++
+.Example
+----
+Old method (encoded sub value):
+g, ChdiZWhuaWEuZkBtdG5pcmFuY2VsbC5pchICYWQ, role:admin
+New method (actual user identifier):
+g, user@example.com, role:admin
+----
++
+--
+link:https://issues.redhat.com/browse/GITOPS-5812[GITOPS-5812]
+--
+
+* Before this update, Argo CD components, such as, `server`, `repo-server`, and `application controller` could crash when accessing the Redis instance due to network or DNS instabilities within the cluster. This issue stemmed from a race condition in the `go-redis` client library when multiple connections in a connection pool call the `dial hook` function.
+With this update, this issue is fixed. This update resolves the issue by integrating an updated go-redis client library that eliminates race conditions during `dial hook` function calls. It also improves the handling and recovery from network and DNS errors, ensuring greater stability for Argo CD components. link:https://issues.redhat.com/browse/GITOPS-6287[GITOPS-6287]
+
+[id="known-issues-1-16-0_{context}"]
+== Known issues
+
+* There is currently a known issue that may prevent successful application sync using impersonation due to insufficient permissions in the openshift-gitops-openshift-gitops-argocd-application-controller ClusterRole. The following error message may appear:
```

Comment:
@varshab1210 - Isn't this fixed in 1.16?

Reply:
Yes, @Dhruv-Soni11 please remove this entry, it was supposed to be added for 1.15 RN as a known issue but we had decided to skip it. Sorry about the confusion. We will not be mentioning about the feature as its still alpha upstream.

Reply:
I have removed this known issue as per your instructions @varshab1210. Thanks.